framework/console/controllers/BaseMigrateController.php

```diff
@@ -32,14 +33,24 @@
      */
     public $defaultAction = 'up';
     /**
-     * @var string the directory containing the migration classes. This can be either
+     * @var string|array the directory containing the migration classes. This can be either
      * a [path alias](guide:concept-aliases) or a directory path.
      *
      * Migration classes located at this path should be declared without a namespace.
      * Use [[migrationNamespaces]] property in case you are using namespaced migrations.
      *
      * If you have set up [[migrationNamespaces]], you may set this field to `null` in order
      * to disable usage of migrations that are not namespaced.
+     *
+     * Since version 2.0.12 you may also specify an array of migration paths that should be searched for
+     * migrations to load. This is mainly useful to support old extensions that provide migrations
+     * without namespace and to adopt the new feature of namespaced migrations while keeping existing migrations.
+     *
+     * In general, to load migrations from different locations, [[migrationNamespaces]] is the preferable solution
+     * as the migration name contains the origin of the migration in the history, which is not the case when
+     * using multiple migration paths.
+     *
+     * @see $migrationNamespaces
      */
     public $migrationPath = '@app/migrations';
```

Comment:
I had to wrap this in brackets, so the CLI can recognize it as an array

    public $migrationPath = ['@app/migrations'];